db/db.json

```diff
@@ -5,14 +5,128 @@
       "firstName": "John",
       "lastName": "Carmack",
       "street": "10 Keen Street",
-      "city": "London"
+      "city": "Amman",
+      "email": "kuhihkjhkjhk@jkjhkjhkj",
+      "linkedIn": "phew",
+      "type": "work",
+      "twitter": "hihihih"
     },
     {
       "id": 2,
       "firstName": "Grace",
       "lastName": "Hopper",
       "street": "32 Deebug Road",
+      "type": "personal",
       "city": "London"
+    },
+    {
+      "firstName": "Omar",
+      "lastName": "Alsyouf",
+      "street": "192, manor lane",
+      "city": "London",
+      "email": "omaralsoyouf@gmail.com",
+      "linkedIn": "phew ",
+      "twitter": "oioioioioioi",
+      "type": "personal",
+      "id": 3
+    },
+    {
+      "firstName": "Omar",
+      "lastName": "Alsyouf",
+      "street": "192, manor lane, manor lane",
+      "city": "London",
+      "email": "omaralsoyouf@gmail.com",
+      "linkedIn": "phew",
+      "twitter": "best guy ever",
+      "type": "work",
+      "id": 4
+    }
+  ],
+  "meetings": [
```

Comment:
Better to have an empty meetings array for every contact which then can be populated.